fts-cron/Dockerfile_cpp

```diff
@@ -1,30 +1,32 @@
-FROM centos:7
+FROM almalinux:9
 
 # Repos needed for (VOMS and FTS) and WLCG certs
 ADD ca.repo /etc/yum.repos.d/ca.repo
+ADD wlcg-el9.repo /etc/yum.repos.d/wlcg-el9.repo
 
-RUN yum install -y epel-release.noarch http://linuxsoft.cern.ch/wlcg/centos7/x86_64/wlcg-repo-1.0.0-1.el7.noarch.rpm && \
-    yum clean all && \
-    rm -rf /var/cache/yum
+# Enable EPEL
+RUN dnf install -y yum-utils \
+  && dnf config-manager --set-enabled crb \
+  && dnf install -y epel-release
 
-RUN yum update -y && \
-    yum upgrade -y && \
-    yum clean all && \
-    rm -rf /var/cache/yum
 
-RUN yum -y install https://repo.ius.io/ius-release-el7.rpm &&\
-    yum install -y httpd python36u-pip python36-mod_wsgi libaio gcc python36-devel.x86_64 mod_ssl openssl-devel.x86_64 python36-m2crypto libnsl.x86_64 patch.x86_64 xrootd-client && \
-    yum clean all && \
-    rm -rf /var/cache/yum
+RUN dnf update -y && \
+    dnf upgrade -y && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf
+
+    RUN dnf -y install -y httpd python-pip python-mod_wsgi libaio gcc python-devel mod_ssl openssl-devel python3-m2crypto libnsl patch xrootd-client && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf
 
 # Install latest kubectl
 RUN curl -o /usr/bin/kubectl -L https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
 RUN chmod +x /usr/bin/kubectl
 
 # Install VOMS and FTS clients for delegating proxies
-RUN yum -y install ca-certificates.noarch lcg-CA ca_* fetch-crl voms-clients-cpp fts-rest-cli \
+RUN dnf -y install ca-certificates.noarch ca_* fetch-crl voms-clients-cpp fts-rest-cli \
```

Comment:
Why not use the provided meta-package instead of a wildcard?

```console
Name         : ca-policy-lcg
Version      : 1.130
Release      : 1
Architecture : src
Size         : 12 k
Source       : None
Repository   : carepo
Summary      : Trust anchors endorsed by the worldwide LHC Computing Grid collaboration
URL          : http://lcg.cern.ch/
License      : Apache 2.0
Description  : This package encodes the policy on Approval of Certificate Authorities,
             : as documented in JSPG Policy Document https://edms.cern.ch/document/428038,
             : as adopted and amended by the authoritative body or bodies for lcg.
             : By this policy it incorporated those trust anchors accredited by
             : the IGTF under the profiles classic,mics,slcs, except those for
             : which lcg has decided to withdraw its endorsement (currently
             : none) and including added authorities (none).
             : 
             : This package is a meta-package, meaning that its purpose is to trigger
             : installation and deprovisioning of other trust anchor packages.
```